### PR TITLE
Added V4 Model - same predictors but different Y

### PR DESCRIPTION
**Pull request by kimaina**
@ibacher these version of the model, uses the same set of predictors to predict y0 - one day default.

### CODE REVIEW

File: IIT-Prediction/prediction-v4-one-day-pred.Rmd [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+
+```
+
+
+
+### Select only relevant predictors and response variable (y0)
+
+
+```{r fig.align='center',warning=FALSE, cache=F, echo=T}
+y='y0' # response variable
```
Comment (kimaina) [inline IIT-Prediction/prediction-v4-one-day-pred.Rmd:169]: @ibacher this RMD is similar to the other one, we've only changed this line. y0 is defined here: https://github.com/AMPATH/clinical-prediction-ml-models/blob/16ef26442bd81a1e10416c92daaf7899d23d6710/IIT-Prediction/helpers.R#L259

File: IIT-Prediction/prediction-v4-one-day-pred.Rmd [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+
+```
+
+
+
+
+## Step 4: Load the Model into the H2O cluster:
+
+```{r fig.align='center',warning=FALSE, cache=F, echo=T}
+ml_model <- h2o.loadModel("model/V4 - 1Day Default Prediction/1_StackedEnsemble_BestOfFamily_1_day_default_prediction")
```
Comment (kimaina) [inline IIT-Prediction/prediction-v4-one-day-pred.Rmd:242]: @ibacher please use this model

**Comment by kimaina** [comment]
Please use the  first model (1_StackedEnsemble_BestOfFamily_1_day_default_prediction)

**Comment by ibacher** [comment]
@kimaina Just to confirm: there are only three models here? (Double-checking since we had quite a few more in previous goes)

**Comment by kimaina** [comment]
> @kimaina Just to confirm: there are only three models here? (Double-checking since we had quite a few more in previous goes)

I only added top 3, we have over 30 or so

**Pull request merged**